ios/RNCCameraRollManager.m

```diff
@@ -333,19 +366,19 @@ static void RCTResolvePromise(RCTPromiseResolveBlock resolve,
 {
   NSArray<NSURL *> *assets_ = [RCTConvert NSURLArray:assets];
   [[PHPhotoLibrary sharedPhotoLibrary] performChanges:^{
-      PHFetchResult<PHAsset *> *fetched =
```

Comment:
I think this change is only formatting change, please remove all changes that are non-functional

Reply:
Are you joking? :D The last two Reviews, you ask me to fix formatting.

Reply:
I meant fix the formatting to match the *current* formatting. This is a change to your formatting style